index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shaker Abady Protfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+    <header>
+        <h2>My Logo</h2>
+        <nav class="flex-nav">☰</nav>
+    </header>
+    <div class="headline">
+        <h2>I’m Mariane Glad to see you!</h2>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="social-title">LET’S CONNECT</p>
+        <ul>
+            <li class="social" ><a href=""><img src="assets/twitter.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Linkedinicon.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector2.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector3.png" alt=""></a></li>
+        </ul>
+        
+    </div>
```

Comment:
@tresorsawasawa for the background should i take the items peace by peace and place them in the HTML or just use a background image and that's it?

Reply:
Hi @shakerAbuDrais 

I suggest you download the entire image and then use it as `background-image` for the whole section. If you take  items peace by peace it will be difficult to you for styling